vmdb/app/models/orchestration_template.rb

```diff
@@ -90,6 +93,16 @@ def validate_content(manager = nil)
     "No #{ui_lookup(:model => 'ExtManagementSystem').downcase} is capable to validate the template"
   end
 
+  def validate_format
+    "Cannot validate format because template type is unknown"
+  end
+
+  def save_with_format_validation
+    msg = draft? ? nil : validate_format
+    raise MiqException::MiqParsingError, msg if msg
```

Comment:
@blomquisg I echoed you earlier comment today about putting more descriptions in the PR to help reviewers understand the reasoning for a new PR. This is a good example. I should have done better.